src/seismometer/_api.py

```diff
@@ -176,9 +176,13 @@ def fairness_audit(metric_list: Optional[list[str]] = None, fairness_threshold=1
     sensitive_groups = sg.cohort_cols
     metric_list = metric_list or ["tpr", "fpr", "pprev"]
 
-    df = pdh.event_score(sg.data(), sg.entity_keys, score=sg.output, summary_method="max")[
-        [sg.target, sg.output] + sensitive_groups
-    ]
+    df = pdh.event_score(
+        sg.data(),
+        sg.entity_keys,
+        score=sg.output,
+        ref_event=sg.predict_time,
+        summary_method=sg.event_combine_strategy(sg.target),
```

Comment:
Since max/min/first/last are aggregate methods and closest to pandas aggregation methods, should we rename these to `aggregation_method`?

More important to me would be can we rename `summary_method` and `sg.event_combine_strategy` as well as the key in config.yml to match?

https://pandas.pydata.org/pandas-docs/version/0.22/generated/pandas.core.groupby.DataFrameGroupBy.agg.html

Reply:
Done in next commit